src/main/java/java/lang/String.java

```diff
@@ -216,11 +237,11 @@ public String(char value[]) {
      *          characters outside the bounds of the {@code value} array
      *
      * @diffblue.limitedSupport
```

Comment:
Why `limitedSupport` now?

Reply:
The reasons are detailed below: count limited by unwind, and value cannot be null.